doc/diffs.md

```diff
@@ -63,26 +65,37 @@ build.sh [-b <BUILD TYPE>] [-f] [-h] [-p] [-t <TARGET>]
 
 ## 50,000 foot view
 
-Assembly code output (aka, "dasm") is generated by running crossgen with a specified JIT to compile a
-specified set of assemblies, by setting the following JIT environment variables to generate
-the output:
+By default, assembly code output (aka, "dasm") is generated by running crossgen
+with a specified JIT to compile a specified set of assemblies, by setting the
+following JIT environment variables to generate the output:
 * `COMPlus_NgenDisasm`
 * `COMPlus_NgenUnwindDump`
 * `COMPlus_NgenEHDump`
 * `COMPlus_JitDiffableDasm`
 * optionally, `COMPlus_NgenGCDump`
 
-Generating "diffs" involves generating assembly code output for both a baseline and a "diff" JIT,
-and comparing the results.
+Generating "diffs" involves generating assembly code output for both a baseline
+and a "diff" JIT, and comparing the results.
+
+Passing the `--pmi` option to `jit-diffs` will instead use reflection to jit each
+method in the assembly, setting these options:
+* `COMPlus_Disasm`
```

Comment:
These variables are wrong (hopefully the code is right). It should be:
```
COMPLUS_JitDisasm
COMPLUS_JitUnwindDump
COMPLUS_JitEHDump
COMPLUS_JitGCDump
```

Reply:
Just typos in the doc, fixed.